### ISSUE TITLE
Wrong barcode number

### ISSUE THREAD

**Issue by bongster** [lindell/JsBarcode#72]
Hi i am using jsbarcode for make a barcode.

when i using jsbarcode, has a problem about show barcode number

i am using v2.5.1

i am send to value '900223824342', but made barcode image is `9002238243422`

so when i update the module, that's problem are solved.

but i wondering what are change from v2.5.1 to 3.x

**Comment by lindell** [comment]
In JsBarcode v2.5.1, the format that was used was automatically chosen based on the best match of barcode symbology if no format was set. Now (v3) I have defaulted to always use CODE128 if available. What happened to you was that `EAN13` was choosen as the barcode and the [checkdigit of EAN13](https://en.wikipedia.org/wiki/International_Article_Number_(EAN)#Calculation_of_checksum_digit) was added to the end of it since it "was missing".
